src/Application/Regulation/Command/VehicleSet/SaveVehicleSetCommand.php

```diff
@@ -31,15 +35,39 @@ public function initFromEntity(?VehicleSet $vehicleSet): self
         $this->otherRestrictedTypeText = $vehicleSet?->getOtherRestrictedTypeText();
         $this->exemptedTypes = $vehicleSet?->getExemptedTypes() ?? [];
         $this->otherExemptedTypeText = $vehicleSet?->getOtherExemptedTypeText();
+        $this->heavyweightMaxWeight = $vehicleSet?->getHeavyweightMaxWeight();
+        $this->heavyweightMaxWidth = $vehicleSet?->getHeavyweightMaxWidth();
+        $this->heavyweightMaxLength = $vehicleSet?->getHeavyweightMaxLength();
+        $this->heavyweightMaxHeight = $vehicleSet?->getHeavyweightMaxHeight();
 
         return $this;
     }
 
-    public function cleanTypes(): void
+    public function clean(): void
     {
         if ($this->allVehicles) {
             $this->restrictedTypes = [];
             $this->otherRestrictedTypeText = null;
+            $this->heavyweightMaxWeight = null;
+            $this->heavyweightMaxWidth = null;
+            $this->heavyweightMaxLength = null;
+            $this->heavyweightMaxHeight = null;
+        }
+
+        if ($this->heavyweightMaxWeight == 0) {
```

Comment:
```suggestion
        if ($this->heavyweightMaxWeight === 0) {
```
C'est mon côté paranoïa sur la comparaison de type.

Reply:
J'ai mis == car je recevais un warning lors du `make check` avec === car la variable est un float

En effet, `0.0` n'est pas `===` à `0`...

Reply:
Si on veut respecter le typage à 100% il faudrait effectivement mettre 0.0`, non ?